types/JudoTypes.tsx

```diff
@@ -21,7 +21,7 @@ export interface JudoConfiguration {
     mobileNumber?: string,
     phoneCountryCode?: string,
     emailAddress?: string,
-    threeDSTwoMaxTimeout?: number,
+    threeDSTwoMaxTimeout?: string,
```

Comment:
Why do we have to revert this change?

Reply:
to only have the changes from `ios/Classes/Wrappers/RNWrappers.m` in the next release